### PR TITLE
fix: add logging for failed fopen operations that don't throw exceptions

### PR DESCRIPTION
**Pull request by miaulalala**
Add logging that shows some more file details when the fopen operation returns false.

### CODE REVIEW

File: lib/private/Streamer.php [hunk 1]
```diff
@@ -126,6 +127,9 @@ public function addDirRecursive(string $dir, string $internalDir = ''): void {
 			if ($file instanceof File) {
 				try {
 					$fh = $file->fopen('r');
+					if($fh === false) {
+						\OCP\Util::writeLog('core', 'unable to open file for stream: ' . print_r($file, true) , ILogger::ERROR);
```
Comment (icewind1991) [inline lib/private/Streamer.php:131]: please use `LoggerInterface` instead of the deprecated `writeLog`

**Review by icewind1991 — CHANGES_REQUESTED** [review]

**Comment by miaulalala** [comment]
Tests are passing on master

**Comment by blizzz** [comment]
What do we do with this PR now?

**Comment by miaulalala** [comment]
> What do we do with this PR now?

Nothing

**Pull request closed without merging**